generation/day_ahead.py

```diff
@@ -160,7 +176,7 @@ def get_series_for_sensor(sensor):
                 belief_time=now,
```

Comment:
Should we change the belief time to something else which is closer to the truth?

Reply:
That is an interesting one. All of the data we are currently querying is published no later than 18:00 on D-1 ([source 1](https://transparency.entsoe.eu/content/static_content/Static%20content/knowledge%20base/data-views/generation/Data-view%20Generation%20Forecast%20-%20Day%20Ahead.html), [source 2](https://transparency.entsoe.eu/content/static_content/Static%20content/knowledge%20base/data-views/generation/Data-view%20Generation%20Forecasts%20-%20Day%20Ahead%20for%20Wind%20and%20Solar.html)). However, in practice we observe that the data is published well before that deadline (and I find it unclear whether the data is eligible for updates before the publication deadline). I reckon the best we could do is to set the belief time for any of these forecasts to `min(now, 6_pm_previous_day)`.

And I agree with you that setting the belief time to now is way off especially when querying historical day-ahead forecasts. So imo this definitely still needs a fix.